modular_skyrat/modules/contractor/code/datums/contractor_items.dm

```diff
@@ -197,3 +197,11 @@
 	item_icon = "comment-slash"
 	limited = 1
 	cost = 2
+
+/datum/contractor_item/baton_upgrade_focus
+	name = "Baton Focus Upgrade"
+	desc = "When applied to a baton, it will exhaust the target even more, should they be the target of your current contract."
+	item = /obj/item/baton_upgrade/focus
+	item_icon = "eye"
+	limited = 1
+	cost = 2
```

Comment:
I'd really need to observe contractors a little more, but I feel like this kind of thing makes the baton feel even more like a one-click win, no?

Like, I get the feeling that this isn't super conducive of roleplay, but then again, I guess you could say the same about contractor in general, as it suffers of mostly the same issues as regular prog traitors.

Reply:
Unless you spend time isolating one person, you are very rarely fighting in a one-on-one, and this is an entire contract's worth of reputation just to make sure that the one person you're targetting is down even more.